### ISSUE TITLE
SOC 2 Report download not working

### ISSUE THREAD

**Issue by aleksdmladenovic** [TraceMachina/nativelink#1389]
In our `Compliance` page, `SOC 2 Report` download is not working.

https://www.nativelink.com/compliance

https://www.nativelink.com/_files/ugd/b37b63_b22d7486c5a14540ab009bdb69fcdda2.pdf

It outputs this.

```
508: Loop Detected (LOOP_DETECTED)

Recursive requests to the same deployment cannot be processed.
```

**Comment by aleksdmladenovic** [comment]
cc: @allada, @MarcusSorealheis, @blakehatch, @SchahinRohani

**Comment by SchahinRohani** [comment]
Fixed with PR #1440
